<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ChangeKeyDialog</class>
 <widget class="QDialog" name="ChangeKeyDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>270</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>560</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>560</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Hauptschlüssel ändern</string>
  </property>
  <widget class="QLabel" name="Banner" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>0</hsizetype>
     <vsizetype>0</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="pixmap" >
    <pixmap/>
   </property>
   <property name="scaledContents" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Q3GroupBox" name="groupBox1" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>80</y>
     <width>540</width>
     <height>150</height>
    </rect>
   </property>
   <property name="title" >
    <string>Schlüssel</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLineEdit" name="Edit_Password" >
    <property name="geometry" >
     <rect>
      <x>200</x>
      <y>30</y>
      <width>310</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Edit_Password_2" >
    <property name="geometry" >
     <rect>
      <x>200</x>
      <y>60</y>
      <width>310</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckBox_Both" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>120</y>
      <width>521</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Passwort UND Schlüsseldatei verwenden</string>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel3" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>90</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>Schlüsseldatei oder Datenträger:</string>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel1_3" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>Passwortwiederholung:</string>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>Passwort:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonChangeEchoMode" >
    <property name="geometry" >
     <rect>
      <x>510</x>
      <y>30</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Browse" >
    <property name="geometry" >
     <rect>
      <x>450</x>
      <y>90</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>W&amp;ählen...</string>
    </property>
    <property name="shortcut" >
     <string>Alt+Ä</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Combo_Dirs" >
    <property name="geometry" >
     <rect>
      <x>201</x>
      <y>90</y>
      <width>240</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="textLabel1" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>56</y>
     <width>450</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text" >
    <string>Geben Sie das Passwort ein und/oder wählen Sie eine Schlüsseldatei.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonCancel" >
   <property name="geometry" >
    <rect>
     <x>457</x>
     <y>240</y>
     <width>90</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text" >
    <string>Abb&amp;rechen</string>
   </property>
   <property name="shortcut" >
    <string>Alt+R</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonOK" >
   <property name="geometry" >
    <rect>
     <x>360</x>
     <y>240</y>
     <width>90</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text" >
    <string>O&amp;K</string>
   </property>
   <property name="shortcut" >
    <string>Alt+K</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends></extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>Edit_Password</tabstop>
  <tabstop>Edit_Password_2</tabstop>
  <tabstop>Combo_Dirs</tabstop>
  <tabstop>Button_Browse</tabstop>
  <tabstop>CheckBox_Both</tabstop>
  <tabstop>ButtonOK</tabstop>
  <tabstop>ButtonCancel</tabstop>
  <tabstop>ButtonChangeEchoMode</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
